valhalla/baldr/graphconstants.h

```diff
@@ -125,14 +125,16 @@ enum class RoadClass : uint8_t {
   kTertiary = 4,
   kUnclassified = 5,
   kResidential = 6,
-  kServiceOther = 7
+  kServiceOther = 7,
+  kRandom = 8
```

Comment:
Unintended change?

Reply:
oh yes! this PR is just my experimental sandbox...not meant for reviewing! :) I'll un-draft it once I have something ready for review.